docs/sources/administration/configuration.md

```diff
@@ -389,6 +389,26 @@ How long the data proxy should wait before timing out. Default is 30 seconds.
 
 This setting also applies to core backend HTTP data sources where query requests use an HTTP client with timeout set.
 
+### keep_alive_seconds
+
+Interval between sends of TCP keepalive requests. Default is 30 seconds.
+
+### tls_handshake_timeout_seconds
+
+The length of time that Grafana will wait for a succussful TLS handshake with the datasource. Default is 10 seconds.
+
+### expect_continue_timeout_seconds
+
+The length of time that Grafana will wait for response headers after initial request headers are written and the request contains the header `Expect: 100-continue`. A value of 0 will result in the body being sent immediately. Default is 1 second.
```

Comment:
one, not 1

Reply:
In most similar cases in the [configuration](https://grafana.com/docs/grafana/latest/administration/configuration) for the default values we have used the number but enclosed it in backquotes (eg. `1`)
So I suggest to change it like this, unless we have recently decided to migrate away from this and I'm not aware about it.

Reply:
This was changed in one of the other suggested changes